consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/MutableBftConfigOptions.java

```diff
@@ -128,9 +142,20 @@ public void setEpochLength(final long epochLength) {
    * @param blockPeriodSeconds the block period seconds
    */
   public void setBlockPeriodSeconds(final int blockPeriodSeconds) {
+    LOG.info("SET BLOCKPERIODSECONDS: " + blockPeriodSeconds);
```

Comment:
Suggest making this (and other log entries) not upper case. And maybe check if this `info` message just comes out once so it's not too noisy being an `info` message.

Reply:
Sure, I had that in mind, I did it this way so it shows very cleraly in the logs since this are the core of the modifications, I'll review it when tests are done.

Reply:
I saw that none of the other methods on that class have any logs, so I removed the log comments for all block period seconds.